src/components/Calculator.js

```diff
@@ -1,89 +1,65 @@
-/* eslint jsx-quotes: ["error", "prefer-single"] */
 import React, { Component } from 'react';
+import Button from './Button';
+import calculate from '../logic/calculate';
+// import operate from '../logic/operate';
 
 class Calculator extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {};
+    this.state = {
+      obj: {
+        total: null,
+        next: null,
+        operation: null,
+      },
+    };
   }
 
   render() {
+    const { obj } = this.state;
+    const handleBtnClick = (buttonName) => {
+      const newObj = calculate(obj, buttonName);
+      this.setState({ obj: newObj });
+    };
+
     return (
-      <div className='calculatorContainer'>
-        <div className='calculatorResult'>0</div>
```

Comment:
- [Optional] Kindly make sure that the calculator result is displayed properly on the screen. The screenshot below is the result of the display right now.
![Screenshot from 2022-09-07 14-25-42](https://user-images.githubusercontent.com/63087491/188890024-d01be87a-9ff5-4ca0-acdf-a755eedc7dea.png)